xml/sle_update_online.xml

```diff
@@ -512,16 +512,16 @@ Continue? [y/n/? shows all options] (y):</screen>
     <screen>&prompt.sudo;zypper repos -u</screen>
     <para>
      In the following you need to rewrite the repository URLs so they point to
-     the respective repositories for the new service pack 
-     (<literal>SLE-15</literal> needs to be replaced with
-     <literal>SLE-15-SP1</literal>). If the URL for a repository looks like
-     this:
+     the respective repositories for the new service pack
+     (<literal>SLE-&product-ga;-SP&product-sp-prev;</literal> needs to be
```

Comment:
If we keep the sentence as it is, the entity &product-sp-prev; does make sense. But can we phrase it differently to avoid mentioning the previous SP? It's an example, we don't know what version the user is on. We should be generic.